creating-elements/readme.md

```diff
@@ -1,58 +1,51 @@
 
-> For example don't hardcode text in the body of the element. Use
-``` html
-<content></content>
-```
-to insert text from a parent.
-The parent can insert the content by:
-``` html
-<child-element> Long text </child-element>
-```
+## 1. Check if there is an element you can use for this already
 
-## Use an already existing element?
-
-> So you've spent at least 5 minutes on a plan
-
-Can you use an existing element?
 Is there an element on the site that has a similar functionality? Can you solve the user story/bugreport by expanding the functionality of this element?
 For example the [paper-input element](https://elements.polymer-project.org/elements/paper-input) is not limited to plain text but can take an argument to restrict the user input to a social security number. Only do this when the it clearly makes sense though!
 
 [The Polymer catalog 💕](https://elements.polymer-project.org/) has a lot of very good elements you can use for free.
 [Customelements.io](https://customelements.io) is also growing quite fast. It's a site where people contribute their code, mostly open source. Not all code is great, but a lot of it is. Also, not all code is Polymer code.
 
-## Create a new element
+## 2. Close your laptop!
+
+Never start writing code right away ⏳. If an [achievable](../glossary/achievable.md) requires you to make a new element, take a deep breath and close your laptop. 
+
+Think about what the function of the element is. Spend some time on it. Imagine future scenarios where the element can be used 🔮. Pinpoint what the element should do, and think carefully about what it shouldn't do.
+
+You are likely to think about building functionality that actually falls outside the scope of a good element. Don't build the element only for the achievable you are working on though, think bigger and make something reusable.🚀
+
+## 3. Create a new element
+
+So you need to build a new element. Think about with which other elements the new element should interact with. What information does the new element need from them and what can it give back? 🗣
+
+Create a HTML file with a name that describes the the element very well and at least one `-`, or hyphen, in it, for instance `user-chip`, `project-name`, `user-login-form` or `helicopter-landing-pad`. 
+
+To get started you could copy and paste the content of `seed-element.html` into your empty file to setup your new element.
 
-So you need to build a new element. Think about with which other elements the new element should interact. What information does the new element need from them and what can it give back? 🗣
+Every element should be able to work in isolation and you should build/test it in isolation. Create a demo file, insert your element, give it the relevant data and test your file.
```

Comment:
maybe explain what a demo file, or link to a description. Now its kinda bomb dropping
.